example/README.md

```diff
@@ -0,0 +1,200 @@
+
+This ID will get stored in the FHIR server database, but it is separate from the IDs used within the FHIR server itself.
+This ID serves to link objects in the FHIR server to their origins in the data table.
+
+For a minimal implementation of `ExamplePatient`, we will not need to implement any other methods. We will discuss
+other methods you can implement later.
+
+## Implementing `get_patient_observations`
+
+The `get_patient_observations` method should get each observation for a given
+patient. Using the `get_identifier_value` method we implemented above, the code below shows how to create
+an Iterable of `ExampleObservation`
+
+```python
+class ExampleDataSource(PatientDataSource):
+
+    ...
+
+    def get_patient_observations(self, patient : Patient):
+        return (ExampleObservation(row) for _, row in self.data[self.data['patient_id'] == int(patient.get_indentifier_value())].iterrows())
+```
+
+Once again, we will need to implement the `__init__` method for `ExampleObservation`.
+
+## Implementing `ExampleObservation`
+
+To implement `__init__`, all we need to do is store the `row` variable passed from `get_patient_observations`:
+
+```python
+class ExampleObservation(Observation):
+
+    ...
+
+    def __init__(self, observation_info):
+        self.observation_info = observation_info
+```
+
+The next two methods we need to implement are `get_observation_type` and `get_value`.
+
+`get_observation_type` returns one of the supported [observation_types.json](../observation_types.json).
+For this small example, we only have `bodyweight`.
+
+```python
+    def get_observation_type(self):
+        return 'bodyweight'
+```
+
+`get_value` returns the actual observed value, in this case the patient weights. The code below is how
+`ExampleObservation` implements `get_value`.
+
+```python
+    def get_value(self):
+        return self.observation_info['body_weight_kg']
+```
+
+Note that the "bodyweight" observation type in [observation_types.json](../observation_types.json) was configured to report the units as "kg".
+Since the body weights in our table are already in kg, we are good. If the weights in our table were in, say, lbs, then we would have to do one of the following:
+- Convert from lbs to kg in our `ExampleObservation.get_value` so that we provide values that match the unit label, or
+- Reimplement `Observation.get_unit_code` with our own `ExampleObservation.get_unit_code` that returns the unit code '[lb_av]' from the [UCUM system](https://ucum.org/). This unit code sometimes looks odd, but you can always implement `ExampleObservation.get_unit_string` to return something human readable like 'lbs'.
+
+These are the last methods we are required to implement; however, there is one last file
+we will need to create.
+
+## Creating JSON config file
+
+In order for `populate_fhir_server.py` to recognize `ExampleDataSource`, we need to create
+a new JSON config file with the proper information. [example.json](example.json) shows the format
```

Comment:
Once the json args like `module_path`, `class_name` are finalized in how they should work (see my other comment on using `module_path`), I think we shouldn't just link to example.json here, but rather we should show the contents and explain `args`, `module_path`, `class_naem`